src/js/pages/organizer/Surveys.jsx

```diff
@@ -41,7 +42,9 @@ export default class Surveys extends React.Component {
   }
 
   loadPreferences = () => {
-    this.setState({ disabled: !PreferenceStore.preferences.survey });
+    this.setState({ disabled: !PreferenceStore.preferences.survey,
```

Comment:
I don't follow. There is currently a new line between attributes, should it all be the same lien?